src/main/java/uk/ac/cam/cl/dtg/segue/api/monitors/PasswordResetByEmailMisuseHandler.java

```diff
@@ -35,16 +35,23 @@ public class PasswordResetByEmailMisuseHandler implements IMisuseHandler {
     
     private static final Logger log = LoggerFactory.getLogger(PasswordResetByEmailMisuseHandler.class);
 
-    private static final Integer SOFT_THRESHOLD = 2;
-    private static final Integer HARD_THRESHOLD = 4;
-    private static final Integer ACCOUNTING_INTERVAL = 6 * NUMBER_SECONDS_IN_MINUTE;
+    private static Integer SOFT_THRESHOLD;
+    private static Integer HARD_THRESHOLD;
+    private static Integer ACCOUNTING_INTERVAL;
     
     /**
      * 
      */
     @Inject
     public PasswordResetByEmailMisuseHandler() {
+        this(2, 4, NUMBER_SECONDS_IN_MINUTE);
+    }
 
+    @Inject
+    public PasswordResetByEmailMisuseHandler(Integer softThreshold, Integer hardThreshold, Integer interval) {
+        this.SOFT_THRESHOLD = softThreshold;
```

Comment:
Since these values are static, you shouldn't need the `this.`

Reply:
Actually, I have a question: since these values are static, how come you're changing them based on the values passed to the constructor?

Reply:
... they probably shouldn't be static actually. While I don't think there should ever be a situation where the application has multiple instances of a misuse handler existing at the same time, these probably don't want to be static any more...